Cargo.toml

```diff
@@ -12,6 +12,7 @@ packed_struct = "0.2"
 packed_struct_codegen = "0.2"
 serde = "1.0"
 serde_derive = "1.0"
+crc-any = "2.3.0"
```

Comment:
I would go with the style of the other deps and only specify major.minor release

Reply:
done